# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../reiserfsprogs/no-coredump.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- reiserfsprogs-3.6.11/lib/misc.c.orig	2003-08-16 11:58:25.000000000 +0200
+++ reiserfsprogs-3.6.11/lib/misc.c	2003-08-16 11:59:32.000000000 +0200
@@ -50,7 +50,7 @@
     va_end (args);
 
     fprintf (stderr, "\n%s\n", buf);
-    abort ();
+    exit (-1);
 }
 
 
